rest/call/list-get-example-4/list-get-example-4.5.x.cs

```diff
@@ -0,0 +1,27 @@
+// Download the twilio-csharp library from twilio.com/docs/csharp/install
+using System;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+
+class Example
+{
+  static void Main(string[] args)
+  {
+    // Find your Account Sid and Auth Token at twilio.com/console
+    const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+    const string authToken = "your_auth_token";
+    TwilioClient.Init(accountSid, authToken);
+
+    var status = CallResource.StatusEnum.InProgress;
+    var startTimeAfter = new DateTime(2009, 07, 04);
+    var endTimeBefore = new DateTime(2009, 07, 06);
+    var calls = CallResource.Read(status: status,
+                                  startTimeAfter: startTimeAfter,
+                                  endTimeBefore: endTimeBefore);
+
+    foreach (var call in calls)
```

Comment:
This one looks good -- another way to do it is:

```csharp
calls.ToList().ForEach(call => Console.WriteLine(call.To));

// or

calls.Select(c => c.Status).ToList().ForEach(Console.WriteLine);
```

I wouldn't change your implementation.